lldb/source/Utility/Logging.cpp

```diff
@@ -49,10 +49,16 @@ static constexpr Log::Category g_categories[] = {
   {{"watch"}, {"log watchpoint related activities"}, LIBLLDB_LOG_WATCHPOINTS},
 };
 
+static constexpr Log::Category g_swift_categories[] = {
+  {{"health"}, {"log all messages related to lldb Swift operational health"}, LIBLLDB_SWIFT_LOG_HEALTH},
+};
```

Comment:
Can you wrap all the code additions in `#ifdef LLDB_ENABLE_SWIFT` or move them to Swift-specific source files?